src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -566,7 +564,9 @@ private static SymbolDisplayPartKind GetPartKind(INamedTypeSymbol symbol)
                 case TypeKind.Submission:
                 case TypeKind.Module:
                 case TypeKind.Class:
-                    return SymbolDisplayPartKind.ClassName;
```

Comment:
It feels like we want to keep this code, simply replacing condition with IsRecord check.